service/health/types.go

```diff
@@ -60,7 +61,7 @@ func NewNodeStatus(node v1.Node) NodeStatus {
 		Hostname:     hostnameFromAddresses(node.Status.Addresses),
 		InstanceType: node.GetObjectMeta().GetLabels()["beta.kubernetes.io/instance-type"],
 		CPUCount:     node.Status.Capacity.Cpu().Value(),
-		Memory:       node.Status.Capacity.Memory().String(),
+		Memory:       nodeMemoryToInt(node.Status.Capacity.Memory()),
```

Comment:
Let's reflect this in the variable name as well - so something like `MemoryKB` maybe?